snc.sh

```diff
@@ -268,11 +271,5 @@ ${OC} patch config.imageregistry.operator.openshift.io/cluster --patch='[{"op":
 # Since this CRD block namespace deletion forever.
 ${OC} delete apiservice v1beta1.metrics.k8s.io
 
-# Get the cert pod name
-cert_pod=$(${OC} get pod -l k8s-app=kubelet-bootstrap-cred-manager -o jsonpath="{.items[0].metadata.name}" -n openshift-machine-config-operator)
-# Remove the bootstrap-cred-manager daemonset
-${OC} delete daemonset.apps/kubelet-bootstrap-cred-manager -n openshift-machine-config-operator
-# Wait till the cert pod is removed
-${OC} wait --for=delete pod/$cert_pod --timeout=120s -n openshift-machine-config-operator
 # Remove the cli image which was used for the bootstrap-cred-manager daemonset
```

Comment:
why do we delete this image?

Reply:
Because this image is extra which we download to rotate the certs.